elements/nuxeo-web-ui-bundle.html

```diff
@@ -705,6 +705,14 @@
   </template>
 </nuxeo-slot-content>
 
+<!-- BULK EDIT -->
+
+<nuxeo-slot-content name="bulkEditResultsAction" slot="RESULTS_SELECTION_ACTIONS" order="10">
```

Comment:
Don't forget to disable the feature:
```suggestion
<nuxeo-slot-content name="bulkEditResultsAction" slot="RESULTS_SELECTION_ACTIONS" order="10" disabled>
```